pkg/apis/core/types.go

```diff
@@ -3197,9 +3206,17 @@ type PodStatus struct {
 	// give the resources on this node to a higher priority pod that is created after preemption.
 	// +optional
 	NominatedNodeName string
+
+	// IP address of the host to which the pod is assigned. Empty if not yet scheduled.
 	// +optional
 	HostIP string
```

Comment:
when pluralizing the singular is obtained from the plural IIRC, it will be good to check how PodIP was pluralized and follow the same steps